### ISSUE TITLE
Config for X, Y position - maybe Mod Menu support

### ISSUE THREAD

**Issue by sternschnaube** [jjblock21/ContainerSearch#1]

**Comment by jjblock21** [comment]
I assume you mean customizing the X and y position of the sear bar for compatibility with other mods.
I might implement this feature in a feature update, but it's a bit lower on my to-do.

I've also researched mod menu integration, but primarily for other settings like the way results are displayed or the search is handled.

**Comment by sternschnaube** [comment]
Yep, that's exactly what I've mean.
Great to hear - thanks 🥳

In the past, on version 1.19, I was using this mod here:
https://modrinth.com/mod/searchable-containers-retraversal

Maybe you could implement the search bar features aswell?
Right-click: clears the search field.
Double-click: highlight the inventory items (if I remember correctly).

**Comment by jjblock21** [comment]
Oh, I wasn't aware of this mods existence.
I'll implement some of those features, but I dont want to copy to much from others.

**Comment by sternschnaube** [comment]
Sounds great 😊
Sadly, his GitHub is not working anymore - at least the linking from Modrinth leads me into nothing.

**Comment by sternschnaube** [comment]
Hello @jjblock21, any update on this? 😊

**Comment by jjblock21** [comment]
Yes, I have started to work on it but am currently busy. You can probably expect an update with all the requested features around Christmas though.

**Comment by sternschnaube** [comment]
Thanks for the fast answer 😄
Sounds amazing, take all your time.

**Comment by sternschnaube** [comment]
Any update? No rush 😊

**Comment by jjblock21** [comment]
I got a little distracted by different projects but the update is still coming.

**Comment by sternschnaube** [comment]
Great!! Thanks for the fast response.

**Comment by sternschnaube** [comment]
Update news @jjblock21? 😄

**Comment by jjblock21** [comment]
Yes, haven't checked whether the mod is broken on 1.20.6 yet but I will update it if it is.

**Comment by jjblock21** [comment]
Looks like there will be no forge version for 1.20.5 and 1.20.6 since architectury currently doesn't support it.

**Comment by sternschnaube** [comment]
I use Fabric personal.
Maybe you check NeoForge out.

**Comment by sternschnaube** [comment]
Any update (specific for Fabric) @jj21asd?

**Comment by Deano7497** [comment]
I'd like an update myself. maybe a beta version if there's anything usable but its fine if not ofc

**Comment by Deano7497** [comment]
hey jj will container search include compatibility for metal bundles and bags of holding on forge?

**Comment by Deano7497** [comment]
any news?

**Comment by Deano7497** [comment]
oh and happy holidays
